s3_storage_client.go

```diff
@@ -38,11 +39,19 @@ type s3Location struct {
 	s3Path     string
 }
 
+// S3LoggingMode allows to configure which logs should be included.
+// By default no logs are included.
+// See https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/aws#ClientLogMode for allowed values.
```

Comment:
Unlikely I guess but let's say a default would change, shouldn't we set it explicitly on our side to protect against different value in the future?

Reply:
The default is on our side - see the line below.